src/desdemona/catalogs/sample_catalog.clj

```diff
@@ -1,5 +1,4 @@
-(ns desdemona.catalogs.sample-catalog
-  (:require [desdemona.functions.sample-functions :refer [transform-segment-shape prepare-rows]]))
+(ns desdemona.catalogs.sample-catalog)
```

Comment:
It does appear to work, so I wonder if `desdemona.functions.sample-functions` is being required somewhere else? The functions aren't ever _called_ in here, I believe they're just referenced to be called later on.